ras_backstage/resources/sample/sample.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from flask import jsonify, make_response, request, Response
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from ras_backstage import sample_api
+from ras_backstage.common.filters import get_collection_exercise_by_period
+from ras_backstage.controllers import collection_exercise_controller, sample_controller, survey_controller
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@sample_api.route('/<short_name>/<period>')
+class Sample(Resource):
```

Comment:
There's a small amount of extra code that can be added here to make the swagger api accept files and (I think) validate that a file is required for this request. There's an example of it in [frontstage-api](https://github.com/ONSdigital/ras-frontstage-api/blob/master/frontstage_api/resources/surveys/upload_collection_instrument.py#L16)
This could also be done for the upload collection instrument